.circleci/config.yml

```diff
@@ -85,10 +85,41 @@ jobs:
               api/src/main/webapp/WEB-INF/sa-key.json
       - run:
           working_directory: ~/workbench/api
-          command: ./project.rb integration
+          command: ./project.rb integration --project all-of-us-workbench-test
+      - save_cache:
+          paths:
+            - ~/.gradle
+            - ~/.m2
```

Comment:
are we using maven?

Reply:
I'm not certain, but we appeared to be caching this on some of the other circle jobs?